src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplevariabledeclarations/Example1.java

```diff
@@ -0,0 +1,24 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MultipleVariableDeclarations"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.multiplevariabledeclarations;
+
+// xdoc section -- start
+public class Example1 {
+  public void myTest() {
+    int mid = 0;
+    int high = 0;
+
+    int lower, higher; // violation, multiple variables declared in same statement
```

Comment:
This Check has two violations https://checkstyle.org/checks/coding/multiplevariabledeclarations.html#Violation_Messages
It is not clear what violation is there, please use part of violation message 
`// violation, 'variable declaration must be in its own statement'`
Or ir something similar

Reply:
Would the following change address the issue? Please let me know if this works or if any further adjustments are needed.

```
int lower, higher;// violation, variables declaration lower, higher
    // must be in its own statement
```

Reply:
You missed single quotes

Reply:
Addressed